docs/index.md

```diff
@@ -2,6 +2,8 @@
 
 [LogDNA](https://logdna.com) is a centralized log management platform. The LogDNA Provider allows organizations to manage Views and Alerts programmatically via Terraform.
 
```

Comment:
I like this :) some of this information is also below (i.e. the default url, servicekey information) so should we remove those parts?

Reply:
Agree that info wouldn't typically need to be repeated.. But I can't find it below. ;/ I think it will be easier after this massive PR is merged. We can polish again.